### PR TITLE
added median function

### PR DESCRIPTION
**Pull request by arkwave**
This function was missing. Added it in to appease the Old Gods. Hail Cthulu.

### CODE REVIEW

File: stats.py [hunk 1]
```diff
@@ -4,3 +4,10 @@
 
 def mean(x):
     raise NotImplementedError()
+
+def median(data):
+	#finds the median value of a given range of values.
+	data.sort()
```
Comment (stefanv) [inline stats.py]: Very creative, but this won't run :)  Change to something that executes!  Feel free to use np.median.

**Comment by AceHao** [comment]
does it returns anything?

**Comment by Christopher-Tennant** [comment]
The median is a better measure of central tendency in skewed distributions.

File: stats.py [hunk 2]
```diff
@@ -4,3 +4,10 @@
...
+	index = len(data)
+	return data(index//2)
+
```
Comment (AceHao) [inline stats.py]: What does this line do.

File: stats.py [hunk 3]
```diff
@@ -1,6 +1,10 @@
...
 This is our brand new stats module
 """
-
+import numpy as np
 def mean(x):
     raise NotImplementedError()
+
+def median(data):
+	#finds the median value of a given range of values.
+	return np.median(data)
```
Comment (stefanv) [inline stats.py]: Perfect, thanks!

**Pull request merged**